From 595201d3910886e323733ea21b5517d7a751b8c1 Mon Sep 17 00:00:00 2001
From: bui duc phuc <phucduc.bui@gmail.com>
Date: Wed, 29 Jul 2026 14:35:32 +0700
Subject: [PATCH 08646/15333] ASoC: sprd: sprd-pcm-dma: Drop redundant error
 messages

The called functions already log failures where appropriate. Remove the
error log here to avoid duplicate error messages.

Signed-off-by: bui duc phuc <phucduc.bui@gmail.com>
Reviewed-by: Baolin Wang <baolin.wang@linux.alibaba.com>
Link: https://patch.msgid.link/20260729073532.56468-2-phucduc.bui@gmail.com
Signed-off-by: Mark Brown <broonie@kernel.org>
---
 sound/soc/sprd/sprd-pcm-dma.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/sound/soc/sprd/sprd-pcm-dma.c b/sound/soc/sprd/sprd-pcm-dma.c
index cbf5bf82d96e..f509a4601de2 100644
--- a/sound/soc/sprd/sprd-pcm-dma.c
+++ b/sound/soc/sprd/sprd-pcm-dma.c
@@ -469,8 +469,6 @@ static int sprd_soc_platform_probe(struct platform_device *pdev)
 
 	ret = devm_snd_soc_register_component(&pdev->dev, &sprd_soc_component,
 					      NULL, 0);
-	if (ret)
-		dev_err(&pdev->dev, "could not register platform:%d\n", ret);
 
 	return ret;
 }
-- 
2.55.0

